pom.xml

```diff
@@ -148,7 +149,22 @@
                     </includes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
+                </configuration>
+            </plugin>
         </plugins>
+        <extensions>
+            <extension>
+                <groupId>com.github.seahen</groupId>
+                <artifactId>maven-s3-wagon</artifactId>
```

Comment:
Remove this

Reply:
Hi Paul

We've encountered a few issues with this feature - just wanted to ensure they're all ironed out before we proceed with the pull request.

Cheers
Egor